esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -323,10 +323,31 @@ void WsWuInfo::sendImportedWorkunitComponentLog(const char* logFile, CHttpRespon
     response->send();
 }
 
+void WsWuInfo::sendErrorMessage(CHttpResponse* response, LogAccessLogFormat format, const char* message)
+{
+    if (format == LOGACCESS_LOGFORMAT_json)
+    {
+        StringBuffer resp;
+        resp.set("{ ");
+        resp.appendf("\"Error\": \"%s\"", message);
```

Comment:
minor: If this is a general purpose function then it should be json-encoding the text (e.g. quoting double quotes) 
Should this be a member of CHttpResponse?  Is it a general function for an http response, or specific?

Reply:
I am going to use the appendJSONStringValue function to get the json right. Also create a CHttpResponse.setErrorLogContent().